src/main/java/Duke.java

```diff
@@ -1,10 +1,81 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printWelcomeMessage();
+        Scanner sc = new Scanner(System.in);
+
+        String phrase = sc.nextLine();
+
+        while (!phrase.equals("bye")) {
+            if (phrase.equals("list")) {
+                Timetable.getTasks();
+            } else if (phrase.contains("done")) {
+                int taskNumber = parseDoneInputToInt(phrase);
+                Timetable.updateTasks(taskNumber);
+            } else {
+                recordTask(phrase);
+            }
+            phrase = sc.nextLine();
+        }
+
+        printFarewellMessage();
+    }
+
+    static void printWelcomeMessage() {
+        String greeting = "Hi! I'm Herrick, your task manager." + "\n"
+                + "What would you like to add to your timetable?";
+        System.out.println(greeting);
+    }
+
+    static void printFarewellMessage() {
+        String farewell = "Bye. Hope you will complete everything for today!";
+        System.out.println(farewell);
+    }
+
+    static void printPutTaskOnList() {
+        String toPrint = "Alright. I'll put it on the list.";
+        System.out.println(toPrint);
+    }
+
+    static void printNumberOfTasks() {
+        if (Timetable.size() <= 1) {
+            System.out.println("For now, you have " + Timetable.size() + " task on the list");
+        } else {
+            System.out.println("For now, you have " + Timetable.size() + " tasks on the list");
+        }
+    }
+
+    static int parseDoneInputToInt(String phrase) {
+        phrase = phrase.replace("done ", "");
+        return Integer.parseInt(phrase);
+    }
+
+    static Timetable parsePhraseToTask(String phrase) {
+        Timetable task;
+
+        if (phrase.contains("todo")) {
+            task = new Todo(phrase.replace("todo ", ""));
+        } else if (phrase.contains("deadline")) {
+            phrase = phrase.replace("deadline ","");
+            String[] whatAndWhen = phrase.split(" /by ");
```

Comment:
`whatAndWhen` may not be very descriptive. can consider using `descriptionAndDeadline`